common/achievement/bucket.ts

```diff
@@ -39,8 +51,20 @@ export const bucketCreatorDefs: BucketCreatorDefs = {
             const { context } = bucketContext;
             // the context.type is a discriminator to define what relationType is used for the bucket (match, subcourse, global_match, global_subcourse)
             // using the context key context[context.type] is equivalent for using a variable key like context.match etc..., meaining that this forEach is iterating over an array of matches/subcourses
-            const matchBuckets = context.match.map(createLectureBuckets).reduce((acc, val) => acc.concat(val), []);
-            const subcourseBuckets = context.subcourse.map(createLectureBuckets).reduce((acc, val) => acc.concat(val), []);
+            const matchBuckets = context.match.map((match) => createLectureBuckets(match, LectureFilter.start)).reduce((acc, val) => acc.concat(val), []);
+            const subcourseBuckets = context.subcourse
+                .map((subcourse) => createLectureBuckets(subcourse, LectureFilter.start))
+                .reduce((acc, val) => acc.concat(val), []);
+            return { bucketKind: 'time', buckets: [...matchBuckets, ...subcourseBuckets] };
+        },
+    },
+    by_lecture_participation: {
+        function: (bucketContext): GenericBucketConfig<TimeBucket> => {
+            const { context } = bucketContext;
+            const matchBuckets = context.match.map((match) => createLectureBuckets(match, LectureFilter.start)).reduce((acc, val) => acc.concat(val), []);
```

Comment:
whats the difference of the bucket creators `by_lecture_start` and `by_lecture_participation`?
We use `LectureFilter.start` in both of them

Reply:
It was just the wrong lecture filter. It was changed.
The difference is that a lectureFilter.start will check the 15 minutes around the lecture start and the lectureFilter.participation checks from start until the end of a meeting.